### PR TITLE
Create 2019-03-18-announcing-new-community-representative.md

### PR DESCRIPTION
**Pull request by darjutak**

### CODE REVIEW

**Comment by darjutak** [comment]
... something, about who @tpolecat even is and why he's an appropriate choice for the board. (I could help draft something, but not today. It could link to his GitHub profile, mention his leadership role in Typelevel (with a link), his well-known open-source libraries such as Doobie and Tut (with links), and his years of helping Scala newcomers daily on Gitter and IRC.)

**I absolutely agree!** 
Thank you for looking through so quickly!

**Comment by SethTisue** [comment]
oh it should also mention his cool science job 🔭✨ :-)

File: _posts/2019-03-18-announcing-new-community-representative.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+category: blog
+title: "Rob Norris joins Scala Center’s Advisory Board as a new Community Representative"
+by: Darja Jovanovic 
+---
+
+On the 6th of March 2019 the Scala Center Advisory Board welcomed Rob Norris as a new member, succeeding Lars Hupel in the Community Representative role, next to Bill Venners.
+
+The Scala Center Team would like to sincerely thank Lars for his great work since the beginning of the Center, 3 years ago, writing and presenting community proposals (see: [SCP-011](https://github.com/scalacenter/advisoryboard/blob/master/proposals/011-debugging-symbols.md); [SCP-013](https://github.com/scalacenter/advisoryboard/blob/master/proposals/013-sbt-migration.md)). 
+
+We are also grateful that Rob accepted our invitation and wish him a great mandate ahead. Keep reading to find out more about the Community Representative role, how to get in touch, share your opinion or submit a proposal.
```
[the remark this answers is not in the document]
Comment (sjrd) [reply _posts/2019-03-18-announcing-new-community-representative.md]: I'm not a native speaker, but I believe "mandate" is appropriate here, isn't it? Maybe a bit too formal, or with a political connotation?

[the remark this answers is not in the document]
Comment (SethTisue) [reply _posts/2019-03-18-announcing-new-community-representative.md]: yes, both of those things, and then also the phrasing is unnatural. it's hard to explain

maybe "we look forward to his participation"? perhaps that misses some part of the intended meaning, but I don't even know what the intended meaning _is_, that's the problem

[the remark this answers is not in the document]
Comment (darjutak) [reply _posts/2019-03-18-announcing-new-community-representative.md]: changed to "looking forward to working together during his time on the board", thanks @SethTisue for suggesting.

**Comment by SethTisue** [comment]
okay, I've added bio stuff on Rob. @tpolecat can you verify that nothing is inaccurate, and that it only embarrasses you just the right amount?

**Comment by tpolecat** [comment]
I am moderately but not severely embarrassed so 👍 thanks!

**Comment by SethTisue** [comment]
@darjutak let's merge, then? are my tweaks okay?

**Comment by darjutak** [comment]
LVeryGTM!

**Comment by SethTisue** [comment]
@darjutak I leave the tweeting to you

**Comment by som-snytt** [comment]
I noticed the telescope is pointed away from the sparkly stars. Maybe tpolecat will clarify that this is accurate but will be fixed in a future release.

I would have asked if the candidate is supposed to be representative _of_ the community or simply _for_ the community, or as Lincoln had it, _of, by, and for_.

On that score, or as Lincoln had it, four score and seven, I would say that tpolecat is aspirational, or as my doobie t-shirt says, "Be the tpolecat you wish to see."

**Comment by SethTisue** [comment]
@fsalvi this isn't showing up on the site — is there a problem with the publishing job...?

**Pull request merged**